spec/ParseFile.spec.js

```diff
@@ -361,6 +361,46 @@ describe('Parse.File testing', () => {
       });
     });
 
+    it('supports files in json that inside of array', async done => {
+      const oldBaseUrl = "http://old-file-url.example.com";
+      const newBaseUrl = "http://new-file-url.example.com";
+      const file = {
```

Comment:
Would you mind to try with a `Parse.File`?

Reply:
@davimacedo I tried with `new Parse.File`. Test is still failing.